docs/version-history/41/index.html

```diff
@@ -105,45 +105,60 @@
-
-</ul>
-
-<h2 id="deprecated">Removed and Deprecated Features</h2>
-
-<ul>
-<li><i>(Deprecated in 4.0, to be removed in 5.0)</i>: <li>The DTD versions of the MusicXML definitions have been deprecated in favor of the XSD versions. This includes the midixml DTD and XSL files, which will not be replaced with XSD versions. (<a href="https://github.com/w3c/musicxml/issues/401">#401</a>)</li>
-</ul>
-
-<h2 id="documentation">Documentation Changes</h2>
-
-<ul>
-<li>Errors in the docs for <code>accordion-middle, circular-arrow, interchangeable, inversion,</code> and <code>voice</code> have been fixed (<a href="https://github.com/w3c/musicxml/issues/496">#496</a>)
-<li>Musical errors in the XML and Image on the Fauré example have been fixed. (<a href="https://github.com/w3c/musicxml/issues/500">#500</a>)</li>
+<h2>Draft: 2025</h2>
+
+<h2 id="schema">Schema Additions</h2>
+
+<h3 id="elements">New Elements</h3>
+
+<ul>
+<li>None so far</li>
+<!-- 4.0 example...  <li>The <a href="../../musicxml-reference/elements/assess/">&lt;assess&gt;</a> element specifies how default assessment should be overridden for individual notes. (<a href="https://github.com/w3c/musicxml/issues/294">#294</a>)</li> -->
+</ul>
+
+<h3 id="attributes">New Attributes</h3>
+
+<ul>
+<li>None so far</li>
+
+<!-- <li>The above attribute has been added to the <a href="../../musicxml-reference/elements/double/">&lt;double&gt;</a> element. (<a href="https://github.com/w3c/musicxml/issues/368">#368</a>)</li> -->
+
+</ul>
+
+<h3 id="values">New Values</h3>
+
+<ul>
+<li>None so far</li>
+<!-- <li>The <a href="../../musicxml-reference/data-types/smufl-accidental-glyph-name/">smufl-accidental-glyph-name</a> type used by the smufl attribute of the <a href="../../musicxml-reference/elements/accidental/">&lt;accidental&gt;</a>, <a href="../../musicxml-reference/elements/accidental-mark/">&lt;accidental-mark&gt;</a>, <a href="../../musicxml-reference/elements/accidental-text/">&lt;accidental-text&gt;</a>, and <a href="../../musicxml-reference/elements/key-accidental/">&lt;key-accidental&gt;</a> elements now allows SMuFL accidental glyphs from the <a href="https://www.w3.org/2021/03/smufl14/tables/medieval-and-renaissance-accidentals.html">Medieval and Renaissance accidentals</a> and <a href="https://www.w3.org/2021/03/smufl14/tables/kievan-square-notation.html">Kievan square notation</a> ranges. (<a href="https://github.com/w3c/musicxml/issues/330">#330</a>)</li> -->
+
+</ul>
+
+<h3 id="values">Changed Attributes/Values</h3>
+<ul>
+<li>The <a href="../../musicxmlreference/data-types/ending-number/">ending-number</a> type used
+    in the <code>number</code> attribute of the
+    <a href="../../musicxml-reference/elements/ending/">&lt;ending&gt;</a>, previously had a regex
+    that implied that empty spaces were different from the empty string.  This has been clarified.
```

Comment:
'Empty spaces' sounds like a pleonasm :slightly_smiling_face: 

I suggest to replace this with 'a sequence of consecutive space characters' or something like that.